nuget/spec/dependabot/nuget/file_parser_spec.rb

```diff
@@ -216,8 +216,7 @@
         end
       end
 
-      context "that is nested" do
-        let(:directory) { "/dir" }
```

Comment:
Could we keep this line as well as revert the next 2 changes that look like this:

``` diff
-     file: "packages.config",
+     file: "dir/packages.config",
```

Part of the PR that made those changes was to verify when an update job was started not in the root directory and I'd like to maintain the test coverage.